src/hotspot/share/opto/compile.cpp

```diff
@@ -5196,7 +5196,14 @@ void Compile::print_method(CompilerPhaseType cpt, int level, Node* n) {
     ss.print(" %d", iter);
   }
   if (n != nullptr) {
-    ss.print(": %d %s ", n->_idx, NodeClassNames[n->Opcode()]);
+    ss.print(": %d %s", n->_idx, NodeClassNames[n->Opcode()]);
+    if (n->is_CallJava()) {
+      CallJavaNode* call = n->as_CallJava();
+      if (call->method() != nullptr) {
+        ss.print(" -");
+        call->method()->print_short_name(&ss);
+      }
+    }
```

Comment:
Suggestion: using `call->_name` instead of `call->method()->print_short_name()` is slightly simpler and more general (should be equivalent for incremental inlining, but will also print stub names, "uncommon trap", etc. when dumping `PHASE_AFTER_ITER_GVN_STEP` graphs on call nodes).
```suggestion
    ss.print(": %d %s", n->_idx, NodeClassNames[n->Opcode()]);
    if (n->is_Call()) {
      CallNode* call = n->as_Call();
      if (call->_name != nullptr) {
        ss.print(" - %s", call->_name);
      }
    }
```

Reply:
Thanks for the suggestion. I think `_name` is only set if there is no attached method. So, we could combine your suggestion with what I've had. What do you think? (see pushed update)

This gives us:

![image](https://github.com/user-attachments/assets/33aad438-432a-4252-8e5b-c75220491bf5)

Reply:
You are right, that looks good, thanks!